db/schema.rb

```diff
@@ -14,6 +14,7 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "hstore"
```

Comment:
This was auto generated when I rebased with upstream and migrated my db. I'll remove it.